infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackDao.kt

```diff
@@ -446,4 +448,37 @@ class LocationTrackDao(
             logger.daoAccess(AccessType.VERSION_FETCH, "fetchOnlyDraftVersions", ids)
         }
     }
+
+    fun fetchVersionsForPublication(
```

Comment:
Näitä linkin kautta hakemisia on nyt aika samannäköisinä muutama. Tein apinoiden olemassaolevaa logiikkaa. Nämä olisi mahdollista korvata hakemalla vain official-puoli ja laskemalla draftit kotlin-koodissa olioista itsestään. Pidin kuitenkin olemassaolevan logiikan koska tässäkin tavassa on puolensa.